client/src/NoZeroFormatter.js

```diff
@@ -7,7 +7,7 @@ export default class NoZeroFormatter extends Component {
   }
   
   render () {
-    var style = "";
+    let style = "";
```

Comment:
You can use a ternary operator here to allow you to use const.

i.e. const style = this.props.value < 0 ? "neg" : "";